webrtc.html

```diff
@@ -1243,6 +1243,14 @@
                         "RTCRtpTransceiver">mid</a></code> value to the mid of
                         the corresponding media description.</p>
                       </li>
+                      <li>
```

Comment:
> 7. If description is set as a local description, then run the following steps for each media description in description that is not yet associated with an RTCRtpTransceiver object:

This has to do with the mid. If a transceiver has beed assigned the mid of a media description, they are associated.

> 1. Let transceiver be the RTCRtpTransceiver used to create the media description.

The media description was created from data in the transceiver. They might not be associated though.

Clarifications to this part should be done in a separate PR I think.